src/posts/tell-me-why.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Tell me Why"
+author: "shreya-ojha"
+date: "3-10-23"
+tags: ["tell", "tell me why", "why", "backstreetboys", "backstreet"]
+img: "posts/shreya/tell-me-why.md"
```

Comment:
img should be ` img: "posts/shreya/tell-me-why.webp"`
instead of  `img: "posts/shreya/tell-me-why.md"`